.github/workflows/pages.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           cache-version: 0 # Increment this number if you need to re-download cached gems
           working-directory: '${{ github.workspace }}/docs'
       - name: Setup Pages
+        if: ${{ github.event_name == 'push' && github.repository == 'GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration' && github.ref_name == github.event.repository.default_branch}}
```

Comment:
I'm checking if this `if` is necessary, after all, because it seems that this action just fetches metadata about a GitHub Pages site. I think the issue is about the concurrency flag above

Reply:
Done